tardis/io/util.py

```diff
@@ -421,3 +421,10 @@ def download_from_url(url, dst):
                 pbar.update(1024)
     pbar.close()
     return file_size
+
+
+def config_iteratation(dictionary):
```

Comment:
Remove this function if it is unused.

Reply:
Otherwise rename to `config_iteration`

Reply:
Sorry that is a spelling mistake from my side 😓

Reply:
Renamed the function to correctly spell `iteration` :)